storybook/src/Heba/atoms/Buttons/BtnTextOnly/BtnTextOnly.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import cn from "classnames";
+export const BUTTON_OPTIONS = {
+  SIZE: {
+    SMALL: "small",
+    MEDIUM: "medium",
+    BIG: "big",
+  },
+  ICON: {
+    LEFT: "left",
+    RIGHT: "right",
+  },
+};
+export const Button = ({ children, size, iconDir }) => {
```

Comment:
can you explain more details i really don't know and please help me

Reply:
This is the default state for text only btn with icon 
![image](https://user-images.githubusercontent.com/44810632/107862925-eef2d080-6e58-11eb-8697-ea2759978b83.png)
the colour used is 00A8E8
on hover, it gets to a darker grade of the accent colour which is: 0092C9.
your button seems to have a different effect, on hover it gets a lighter grade of the primary colour.
![image](https://user-images.githubusercontent.com/44810632/107862954-26617d00-6e59-11eb-9d51-5ea1571232bc.png)
hope this would help

Reply:
0092C9 not in config and i already use it but not appear
![New Bitmap Image](https://user-images.githubusercontent.com/68795300/107866310-5a03cd80-6e80-11eb-83c0-fe7765c608d7.jpg)

![New Bitmap Image](https://user-images.githubusercontent.com/68795300/107866276-04c7bc00-6e80-11eb-9761-596ae5cb0e77.jpg)